example/android/app/build.gradle

```diff
@@ -100,6 +100,6 @@ android {
 
 dependencies {
     compile "com.android.support:appcompat-v7:23.4.0"
-    compile "com.facebook.react:react-native:0.29.0"  // From node_modules
+    compile "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
This is obviously up to you folks, but I'd make the example app match whatever `react-native init` does, to ensure this works regular apps out of the box.

Good point on the dependency! So if react-native-maps is on 0.29 my app can be on that version or any newer? Right now I'm still on 0.28, so that's why I initially thought the `+` would be nice.